adds importing sequence data tutorial

gregcaporaso opened the pull request:
fixes #33 

I debated merging this tutorial with the existing *Importing* tutorial, but I see this one expanding a lot as we support additional input formats so it seems to make more sense on its own. Any objections to that?

jairideout commented:
I think going with a separate tutorial is a good idea, at least for now. We'll need to think about how to generally (automatically?) document the different semantic types and their supported file formats, as those all apply to importing and exporting. For now, this tutorial will help users move forward and we can always break links and reorganize things in the future since every link to the docs is versioned! 👌

gregcaporaso commented on source/tutorials/import-sequence-data.rst:
```diff
@@ -0,0 +1,56 @@
+
+.. command-block::
+
+   qiime tools import --type RawSequences --input-path raw-sequences/ --output-path raw-sequences
```
Thanks. This one actually works if you include the `.qza` extension. That's what we're doing in the *Moving Pictures* tutorial, just made that change here (I think I must have edited it in parallel with testing yesterday, thinking that the extension wasn't necessary).

gregcaporaso commented:
Thanks for the review @thermokarst! I've addressed all of your comments, build the docs locally, and tested all of the commands by copy/pasting from the built docs. Should be good to go.
